README.md

```diff
@@ -114,3 +114,20 @@ The `Package` class should provide access to the full dependency tree.
 ```sh
 git submodule init
 ```
+
+
+## Registry URI 
+
+The URI to lookup a package from a registry should follow the following format. (subject to change as the Registry Contract Standard makes it's way through the EIP process)
+
+```
+scheme://authority/package-name/version
+```
+
+* URI must be a string type
+* `scheme`: `ercxxx` 
+* `authority`: Must be a valid ENS domain or a valid checksum address pointing towards a registry contract.
+* `package-name`: Must conform to the package-name as specified in the [EthPM-Spec](http://ethpm-spec.readthedocs.io/en/latest/package-spec.html#package-name).
+* `version`: The URI escaped version string.
```

Comment:
Ok, responding to your other comment here about *why*.

1. We want to be flexible to allow alternate version schemes, while probably only supporting semver for anything other than exact version matching.
2. This means this spec needs to be expanded a bit to allow specifying things like `>=1.0,<2` (which I believe is *either* valid semver, or a valid semver abstraction.

So for example (without proper url escaping)

- `ercXXX://packages.augur.eth/safe-math/>=1.0.2,<2`

When I say expanded, I mean we should define the term "version string" loosely and put some sort of recommendation that tooling support this style of semver.  I'd recommend looking for an RFC somewhere about this, and just pointing to that.